cmd/build.go

```diff
@@ -12,11 +12,22 @@ import (
 	"github.com/elastic/elastic-package/internal/docs"
 )
 
+const buildLongDescription = `Use this command to build a package. Currently it supports only the "integration" package type.
+
+Built packages are stored in the "build/" folder located at the root folder of the local Git repository checkout that contains your package folder. The command will also render the README file if there is a corresponding template file present in "_dev/build/docs/README.md". All "_dev" directories will be omitted.
```

Comment:
Maybe say where the generated README file can be found - in your package folder.

Also, I think you can either omit the last sentence about the `_dev` directories being omitted or expand it to say that all `_dev` folders under your package folder will be omitted from the built package.

Reply:
Fixed